packages/ui/components/avatar/Avatar.tsx

```diff
@@ -51,6 +52,7 @@ export function Avatar(props: AvatarProps) {
             {gravatarFallbackMd5 && (
               <img src={defaultAvatarSrc({ md5: gravatarFallbackMd5 })} alt={alt} className={rootClass} />
             )}
+            {size === "lg"}
```

Comment:
What does this do?

Reply:
kept the team icon a proportion up from the avatars in the blocks